tools/mo/unit_tests/mo/utils/freeze_placeholder_test.py

```diff
@@ -124,180 +122,185 @@ def tearDown(self):
+        test_cases = [
             (
-                    "in1[1 4]{f32}->[1.0 2.0 3.0 4.0],in2[1 4]{f32}->[1.0 2.0 3.0 4.0]",
-                    True,
-                    {},
-                    np.array([2.0, 4.0, 6.0, 8.0]),
-                    np.float32,
+                "in1[1 4]{f32}->[1.0 2.0 3.0 4.0],in2[1 4]{f32}->[1.0 2.0 3.0 4.0]",
+                True,
+                {},
+                np.array([2.0, 4.0, 6.0, 8.0]),
+                np.float32,
             ),
             (
-                    "in2{f32}->[0.0 0.0 0.0 0.0]",
-                    True,
-                    {"in1": np.array([[1.0, 2.0], [3.0, 4.0]])},
-                    np.array([[1.0, 2.0], [3.0, 4.0]]),
-                    np.float32,
+                "in2{f32}->[0.0 0.0 0.0 0.0]",
+                True,
+                {"in1": np.array([[1.0, 2.0], [3.0, 4.0]])},
+                np.array([[1.0, 2.0], [3.0, 4.0]]),
+                np.float32,
             ),
             (
-                    "in2{f32}->[1.0 15.0 15.5 1.0]",
-                    True,
-                    {"in1": np.array([[2.0, 4.0], [12.0, 8.0]])},
-                    np.array([[3.0, 19.0], [27.5, 9.0]]),
-                    np.float32,
+                "in2{f32}->[1.0 15.0 15.5 1.0]",
+                True,
+                {"in1": np.array([[2.0, 4.0], [12.0, 8.0]])},
+                np.array([[3.0, 19.0], [27.5, 9.0]]),
+                np.float32,
             ),
             (
-                    "in1[1 4]{i32}->[1 2 3 4],in2[1 4]{i32}->[1 2 3 4]",
-                    True,
-                    {},
-                    np.array([2.0, 4.0, 6.0, 8.0]),
-                    np.int32,
+                "in1[1 4]{i32}->[1 2 3 4],in2[1 4]{i32}->[1 2 3 4]",
+                True,
+                {},
+                np.array([2.0, 4.0, 6.0, 8.0]),
+                np.int32,
             ),
-        ],
-    )
-    def test_freeze_placeholder_with_value_onnx_fe(self, input_freezing_value, use_new_fe, inputs, expected,
-                                                   dtype=None):
+        ]
+
         with patch("openvino.tools.mo.convert_impl.get_default_frontends") as default_fe:
             default_fe.return_value = get_test_default_frontends()
-            args = base_args_config(use_new_fe=use_new_fe)
-            args.input_model = "test_model.onnx"
-            args.input = input_freezing_value
-
-            _, model = prepare_ir(args)
-
-            ie = Core()
-            exec_net = ie.compile_model(model, "CPU")
-            req = exec_net.create_infer_request()
-            results = req.infer(inputs)
-            values = list(results.values())[0]
-            if dtype is not None:
-                assert values.dtype == dtype
-            assert np.allclose(values, expected)
-
-    @generate(
-        *[
+
+            for idx, (input_freezing_value, use_new_fe, inputs, expected, dtype) in enumerate(test_cases):
```

Comment:
Please fix it here as well

Reply:
Done this file comes under special test case thats why i have reverted it back to original one , I have already  created new PR for this one https://github.com/openvinotoolkit/openvino/pull/20313